conda-recipes/D47188-svml.patch

```diff
@@ -560,30 +533,46 @@ index 37ffac1..8ad2131 100644
      // This convention allows using the Win64 convention on other targets.
      case CallingConv::Win64:
 diff --git a/lib/Transforms/Vectorize/LoopVectorize.cpp b/lib/Transforms/Vectorize/LoopVectorize.cpp
-index 5bcf0c0..0c4f666 100644
+index 5bcf0c0..cef0009 100644
 --- a/lib/Transforms/Vectorize/LoopVectorize.cpp
 +++ b/lib/Transforms/Vectorize/LoopVectorize.cpp
-@@ -4917,6 +4917,7 @@ void InnerLoopVectorizer::widenInstruction(Instruction &I) {
+@@ -3974,6 +3974,17 @@ static unsigned getVectorCallCost(CallInst *CI, unsigned VF,
+   if (!TLI || !TLI->isFunctionVectorizable(FnName, VF) || CI->isNoBuiltin())
+     return Cost;
+ 
++  // this goes against LLVM coding philosophy, but it'll stop bleeding
++  bool IgnoreMe;
++  StringRef FuncName = TLI->getVectorizedFunction(FnName, VF, IgnoreMe, true);
++#if LLVM_ON_WIN32
++  StringRef SvmlPrefix("\0_svml", 6); // nobody knows why symbols are like this
```

Comment:
Should `SvmlPrefix` be used in the comparison in the condition below i.e. `FuncName.startswith(SvmlPrefix)` ?

Reply:
ah, sharp eye! thanks! it's bad copy-paste. Looks like the llvmdev test (in conda-forge) doesn't work as intended as well since it didn't catch it